### ISSUE TITLE
NTR: Temporal artery biopsy: giant cells (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9092]
New term request Temporal artery biopsy: giant cells (segal_070319224056):   UMLS: 2237973

**Comment by pnrobinson** [comment]
@MickeySegal
I am wondering how much granularity we require. The giant cells do not occur in isolation, and here is a description of other changes seen on temporal artery biopsy - https://www.frontiersin.org/articles/10.3389/fmed.2022.1066503/full
also
The classic histological picture of GCA is a transmural inflammatory infiltrate associated with marked disruption of the internal elastic lamina and the presence of giant cells. (https://academic.oup.com/rheumatology/article/59/Supplement_3/iii5/5826897?login=false)

thoughts?

**Comment by MickeySegal** [comment]
Our ophthalmology curator also used the finding for https://pubmed.ncbi.nlm.nih.gov/16138997/, but I don't have access to the article to see the bundle of findings there.

It seems best not to describe a bundle of findings, but instead focus on a finding that can be readily assessed pathologically.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000618
- label: Temporal artery giant cells
- parents: [Abnormal superficial temporal artery morphology (HP:0430104)](https://hpo.jax.org/app/browse/term/HP:0430104)
- pmid: PMID:37787225
- def: The presence of giant cells in the temporal artery. This is a feature of giant cell arteritis. Giant cells are multinucleated cells formed by fusion of multiple macrophages and are typically located in the intima of the arterial wall. This feature is typically demonstrated by temporal artery biopsy but other arteries may also be affected.
- comment: See Figure 1 of PMID:37787225.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9092
- synonyms:
